internal/commands/fetch.go

```diff
@@ -65,8 +66,9 @@ func FetchThingByName(fn *FetchName, remote remotes.Remote) ([]byte, error) {
 		}
 	}
 
-	// TODO: how to know if it is official?
-	tmid, err := model.ParseTMID(id, false)
+	official := internal.Prep(tocThing.Author.Name) == internal.Prep(tocThing.Manufacturer.Name)
```

Comment:
I wonder if calling `internal.Prep()` is necessary here.
As a side-note, "Prep" should really be renamed. Note that push [uses](https://github.com/web-of-things-open-source/tm-catalog-cli/blob/ddfa9c111fc6153780897bc83fe99bbfdca419b8/internal/commands/validate/validate.go#L72) `sanitize.BaseName()`

Reply:
On my TODO. I don't think `sanitize.BaseName()` serves the same purpose, though. `Prep()` trims and lower cases strings to make them comparable, e.g. it is used when filtering.

On the naming: util.ToTrimmedLower(s string) string?